game/templates/easy-question.html

```diff
@@ -29,20 +29,17 @@
     <input type="text" name="textfield" id="input" maxlength="{{ans_length}}" placeholder="{{ans_length}} letters" size="12" autofocus />
     <input type="submit" name="button" id="sub_btn" value="Submit"/>
 </form>
-{% if question.is_pass %}
+{% if user.current_easy > question.id %}
     <p><i class="fa fa-check"></i>  You got it!!</p>
+    {% if question.id < 15 %}
+        <button><a href="{% url 'game:e-picture' next_question.id %}">Next</a></button>
+    {% else %}
+        <button><a href="{% url 'game:easy' %}">Next</a></button>
+    {% endif %}
 {% else %}
     <p id="status"><i class="fa fa-close"></i>  Try again</p>
```

Comment:
Try again will always appear if user still not answer. You should try to make it appear only when user put their answer and submit. ❤️

Reply:
I will fix this after we merge with hint features.
Thank you for review 🙇‍♀